### PR TITLE
fluent-cat: Add --time-as-integer option for v0.12 or earlier versions

### PR DESCRIPTION
**Pull request by repeatedly**
For testing, re-install v0.12 is hard for users.

### CODE REVIEW

**Comment by repeatedly** [comment]
@tagomoris How about this option?

File: lib/fluent/command/cat.rb [hunk 1]
```diff
@@ -123,7 +128,7 @@ def run
     end
   end
 
-  def initialize(tag, connector)
+  def initialize(tag, connector, time_as_integer)
```
Comment (tagomoris) [inline lib/fluent/command/cat.rb]: It's better to use keyword arguments for possible further options.

**Comment by repeatedly** [comment]
applied review.

**Comment by tagomoris** [comment]
LGTM

**Pull request merged**